setup.cfg

```diff
@@ -1,16 +1,63 @@
+[metadata]
+name = pydoctor
+version = attr: pydoctor._setuptools_version
+author = Michael Hudson-Doyle
+author_email = micahel@gmail.com
+maintainer = Maarten ter Huurne
+maintainer_email = maarten@boxingbeetle.com
+description = API doc generator.
+long_description_content_type = text/x-rst
+license = MIT/X11
+url = https://github.com/twisted/pydoctor
+project_urls =
+    Issue Tracker=https://github.com/twisted/pydoctor/issues
+    Repository=https://github.com/twisted/pydoctor
+classifiers =
+    Development Status :: 6 - Mature
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+    Topic :: Documentation
+    Topic :: Software Development :: Documentation
+
+[options]
+packages = find:
+setup_requires = incremental
+python_requires = >=3.6
+install_requires =
+    incremental
+    appdirs
+    CacheControl[filecache]
+    Twisted
+    requests
+    astor
```

Comment:
Please add `attrs`: that was missing in `setup.py` as well, but since pydoctor imports `attrs` directly, it should declare it and not rely on getting it indirectly via Twisted.